modules/planning/scenarios/util/util.cc

```diff
@@ -257,15 +259,13 @@ bool CheckADCReadyToCruise(Frame* frame,
   // check gear status
   // TODO(SHU): align with vehicle parameters
   static constexpr double kMinSpeed = 0.1;  // m/s
-  if ((common::VehicleStateProvider::Instance()->gear() ==
-           canbus::Chassis::GEAR_DRIVE ||
-       std::fabs(common::VehicleStateProvider::Instance()
-                     ->vehicle_state()
-                     .linear_velocity()) < kMinSpeed) &&
-      !is_near_front_obstacle && heading_align_w_reference_line) {
-    return true;
-  }
-  return false;
+  return ((common::VehicleStateProvider::Instance()->gear() ==
+               canbus::Chassis::GEAR_DRIVE ||
+           std::fabs(common::VehicleStateProvider::Instance()
```

Comment:
Save `common::VehicleStateProvider::Instance()` to a local var.

Reply:
fixed. Thx!